MOBILE_API.xml

```diff
@@ -9153,6 +9159,13 @@
         </param>
     </function>
     
+    <function name="OnSeekMediaClockTimer" functionID="OnSeekMediaClockTimerID" messagetype="notification" since="7.1">
```

Comment:
Same as before, and please add more space before <description>See StartTime.</description>